packages/recommend/src/types/TrendingFacetHit.ts

```diff
@@ -1,5 +1,5 @@
-export type TrendingFacetHit<TObject> = {
+export type TrendingFacetHit = {
```

Comment:
if this type is exported, it's a breaking change, someone who would be using `TrendingFacetHit<string>` will get a typescript error. Instead all generics should stay, defaulted to `string`

Reply:
IMO it is fine to have a build step breaking change in this case as it is a bug fix in typings.

Plus the usage for this model is very small and the update will be a one liner documented in the release notes.